### ISSUE TITLE
Add max players variable

### ISSUE THREAD

**Issue by danvsel** [Weissnix4711/bdsx-pterodactyl#1]
Would really appreciate if you could add a max players variable to the egg.

**Comment by Weissnix4711** [comment]
Does this shit even still work?

Try this: [egg-b-d-s-x2-rc1.2.json](https://github.com/Weissnix4711/bdsx-pterodactyl/files/14665715/egg-b-d-s-x2.json)

**Comment by danvsel** [comment]
![image](https://github.com/Weissnix4711/bdsx-pterodactyl/assets/106025492/8a1ad8a8-a430-4d33-b33b-b8b2ee8e655d)
![image](https://github.com/Weissnix4711/bdsx-pterodactyl/assets/106025492/c206ac7d-95c3-4b6b-b183-fa438d2a9a15)
Doesnt seem like it works due to the startup command

**Comment by danvsel** [comment]
![image](https://github.com/Weissnix4711/bdsx-pterodactyl/assets/106025492/2ddb28d4-c498-4bde-b29e-fc622dab05f9)
Okay so i fixed the startup issue, but well, the max players thing doesnt work i think

**Comment by Weissnix4711** [comment]
That's just the tests being weird. Either ignore the tests or remove the line `import "./example_and_test";` from the `index.ts` file.

**Comment by danvsel** [comment]
yeah yeah, i removed these things, and the max player var worked. The thing is, i cant join at all. everything seems to work fine and i get no errors, but i just get disconnected from the world
